src/Vts.Scripting/Program.cs

```diff
@@ -0,0 +1,29 @@
+﻿// This is a top-level program that lets you run any of the demos in the Vts.Scripts project.
+// It is a good place to start if you want to run a demo, or if you want to write your own code.
+using Vts.Scripting;
+
+// to run a single demo, find the one you want by browsing to them, and run it:
+Vts.Scripting.ShortCourse.Demo02AnalogVsContinuousWithReflectance.RunDemo();
```

Comment:
Why do you have the using and the fully qualified path here? Can we remove Vts.Scripting?

Reply:
So that when people uncomment the code at the bottom, it will work.

Reply:
I meant can we replace:

`Vts.Scripting.ShortCourse.Demo02AnalogVsContinuousWithReflectance.RunDemo();`

with:

`ShortCourse.Demo02AnalogVsContinuousWithReflectance.RunDemo();`

Reply:
Ah, left that in to match the others - MonteCarlo and ForwardSolvers have collisions without the fully qualified name.